free-courses-en.md

```diff
@@ -210,6 +213,7 @@
 
 ### Ruby
 
+* [Learn Ruby the Hard Way](http://learnrubythehardway.org/book/)
```

Comment:
Good, thanks @switchhax. There no point in duplicating content. @MathewtheCoder could you remove this link?